controllers/state_of_the_world.go

```diff
@@ -130,16 +133,42 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 		// TODO: add tls policy specific tasks to workflow
 	}
 
+	ok, err = openshift.IsConsolePluginInstalled(manager.GetRESTMapper())
+	if err != nil || !ok {
+		logger.Info("console plugin is not installed, skipping related watches and reconcilers", "err", err)
+	} else {
+		controllerOpts = append(controllerOpts,
```

Comment:
think it makes sense to show the kuadrant `ConsolePlugin`, but noticed from the topology screenshot it's showing _all_ `ConsolePlugin` resources, which we probably don't want (e.g. the monitoring-plugin, which isn't related to Kuadrant).

Reply:
Right, I will implement two "filters" in that regards:
* the consoleplugin watcher now contains a predicate with a label selector
* Only watch on the expected namespace, that the operator knows at this stage.

Reply:
perfect